### PR TITLE
Add Ticket Buying Platform

### PR DESCRIPTION
**Pull request by HassanBahati**
- add quicket link
- refactor label for easy maintence

### CODE REVIEW

File: components/Navbar.js [hunk 1]
```diff
@@ -21,8 +21,9 @@ export default function Navbar() {
       label: "Contact",
     },
   ];
-  const QUICKET_LINK = "#";
+  const QUICKET_LINK = "https://qkt.io/w31vDE";
   const SESSIONIZE_LINK = "https://sessionize.com/pycon-uganda";
+  const TICKET_LABEL = "Get Your Ticket";
```
Comment (HassanBahati) [inline components/Navbar.js:26]: About this label...do ya'll think we are good with the wording `Get Your Ticket`

Comment (basiima) [reply components/Navbar.js]: Personally, I'm fine with just `Get Ticket`

**Review by basiima — COMMENTED** [review]
Thanks for the good work @HassanBahati. Maybe the label can be just `Get Ticket`

File: components/Navbar.js (hunk 1 above)
Comment (HassanBahati) [reply components/Navbar.js]: @basiima this is noted and resolved via [ce73449](https://github.com/PyconUg/ug.pycon.org/pull/45/commits/ce73449799c0f1de000c11cbf4ad49827ad82520)

File: components/Navbar.js [hunk 2]
```diff
@@ -21,8 +21,9 @@ export default function Navbar() {
       label: "Contact",
     },
   ];
-  const QUICKET_LINK = "#";
+  const QUICKET_LINK = "https://qkt.io/w31vDE";
   const SESSIONIZE_LINK = "https://sessionize.com/pycon-uganda";
+  const TICKET_BTN_LABEL = "Get Ticket";
```
Comment (Amanya-Brian) [inline components/Navbar.js:26]: This label is just fine

**Review by Amanya-Brian — APPROVED** [review]

File: components/Navbar.js (hunk 2 above)
Comment (HassanBahati) [reply components/Navbar.js:26]: alright

**Pull request merged**